sdk/eventhubs/azure-messaging-eventhubs-stress/README.md

```diff
@@ -0,0 +1,300 @@
+
+```shell
+# Authenticate to Azure
+az login
+
+# Download the kubeconfig for the cluster (creates a 'context' named 'stress-test')
+az aks get-credentials --subscription "Azure SDK Developer Playground" -g rg-stress-cluster-test -n stress-test
+```
+
+### Deploy to cluster
+
+Build out the jar package:
+
+```shell
+cd <current project path>
+mvn clean install
+```
+
+Update the `namespace` field in `Chart.yaml` file. It is suggested to use your own alias to avoid conflict.
+
+```yaml
+name: <stress test name>
+...
+annotations:
+  namespace: <stress test namespace>
+```
+
+Note: the default value of stress test namespace is `java-eh`. You can keep it when no other stress test are running.
+
+Download project [azure-sdk-tools][azure_sdk_tools] for the deployment to stress test cluster.
+
+Keep in current project path, run below command:
+
+```shell
+<root path>\azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 -PushImage -Login -Namespace <stress test namespace>
+```
+
+### Validate status
+
+Get pods and jobs:
+
+```shell
+helm list -n <stress test namespace>
+kubectl get pods -n <stress test namespace>
+kubectl get jobs -n <stress test namespace>
+```
+
+List stress test pods:
+
+```shell
+kubectl get pods -n <stress test namespace> -l release=<stress test name>
+```
+
+Get logs from the init-azure-deployer init container, if deploying resources. Omit `-c init-azure-deployer` to get main container logs.
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name> -c init-azure-deployer
+```
+
+If above command output is empty, there may have been startup failures:
+
+```shell
+kubectl describe pod -n <stress test namespace> <stress test pod name>
+```
+
+Get stress test logs:
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name>
+# Note that we may define multiple containers (for example, sender and receiver)
+kubectl logs -n <stress test namespace> <stress test pod name> -c <container name>
+```
+
+Stop and remove deployed package:
+
+```shell
+helm uninstall <stress test name> -n <stress test namespace>
+```
+
+### Configure Faults
+
+[Chaos Mesh](https://chaos-mesh.org/) is used to configure faults against test jobs. There are two ways for the configuration, which are via the UI or via kubernetes manifests.
+
+#### Chaos Dashboard
+
+Make sure you can access the chaos dashboard by running the below command, and navigating to localhost:2333 in your browser.
+
+```shell
+kubectl port-forward -n stress-infra svc/chaos-dashboard 2333:2333
+```
+From dashboard, click `New experiment` and choose your parameters to submit a fault experiment.
+
+#### Chaos Manifest
+
+See [Chaos manifest](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/chaos/README.md#chaos-manifest) for details.
+
+
+### Configure Monitor
+
+Download [application insights java agent][java_agent_jar]. 
+
+Place the `applicationinsights-agent-3.2.11.jar` under `src\main\resources` folder.
```

Comment:
Can we add this step as part of the Dockerfile?

Reply:
Good ideal. I will try this way.